doorstop/cli/main.py

```diff
@@ -74,7 +74,9 @@ def main(args=None):  # pylint: disable=R0915
     }
 
     # Build main parser
-    parser = argparse.ArgumentParser(prog=CLI, description=DESCRIPTION, **shared)
+    parser = argparse.ArgumentParser(  # type: ignore
+        prog=CLI, description=DESCRIPTION, **shared
+    )
```

Comment:
I am not sure why `mypy` emits a warning for this, but this code is clearly working, so I've disabled it.